packages/InteractiveProfilingTool-Tests.package/IPTModelTestCase.class/instance/testBuildWindow.st

```diff
@@ -0,0 +1,4 @@
+as yet unclassified
+testBuildWindow
+
+	self assert: 'PluggableWindowSpec' equals: self buildWindow className
```

Comment:
Instead of testing if the method returns a spec, this should test if the builder is able to build a `PluggableWindowMorph` using the spec without errors.